affectedmoduledetector/src/main/kotlin/com/dropbox/affectedmoduledetector/AffectedModuleConfiguration.kt

```diff
@@ -41,7 +42,7 @@ class AffectedModuleConfiguration {
      * @see CustomTask - Implementation class
      * @see AffectedModuleDetectorPlugin - gradle plugin
      */
-    var customTasks = emptySet<AffectedModuleConfiguration.CustomTask>()
```

Comment:
The `AffectedModuleConfiguration` qualifier is redundant so the IDE was whinging at me so I removed the qualifer.